drivers/clock_control/clock_control_r8a779f0_cpg_mssr.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ * Copyright (c) 2023 IoT.bzh
+ *
+ * r8a779f0 Clock Pulse Generator / Module Standby and Software Reset
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_r8a779f0_cpg_mssr
+
+#include <errno.h>
+#include <zephyr/arch/cpu.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/renesas_cpg_mssr.h>
+#include <zephyr/dt-bindings/clock/renesas_cpg_mssr.h>
+#include <zephyr/dt-bindings/clock/r8a779f0_cpg_mssr.h>
+#include <zephyr/irq.h>
+#include "clock_control_renesas_cpg_mssr.h"
+
+#define LOG_LEVEL CONFIG_CLOCK_CONTROL_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(clock_control_rcar);
+
+struct r8a779f0_cpg_mssr_cfg {
+	DEVICE_MMIO_ROM; /* Must be first */
+};
+
+struct r8a779f0_cpg_mssr_data {
+	struct rcar_cpg_mssr_data cmn; /* Must be first */
+};
+
+/* NOTE: the array MUST be sorted by module field */
+static struct cpg_clk_info_table core_props[] = {
+	RCAR_CORE_CLK_INFO_ITEM(R8A779F0_CLK_S0D12_PER, RCAR_CPG_NONE,
+				RCAR_CPG_NONE, RCAR_CPG_KHZ(66660)),
+
+	RCAR_CORE_CLK_INFO_ITEM(R8A779F0_CLK_CL16M, RCAR_CPG_NONE,
+				RCAR_CPG_NONE, RCAR_CPG_KHZ(16660)),
+};
+
+/* NOTE: the array MUST be sorted by module field */
+static struct cpg_clk_info_table mod_props[] = {
+	RCAR_MOD_CLK_INFO_ITEM(702, R8A779F0_CLK_S0D12_PER),
+	RCAR_MOD_CLK_INFO_ITEM(704, R8A779F0_CLK_S0D12_PER),
+
+	RCAR_MOD_CLK_INFO_ITEM(915, R8A779F0_CLK_CL16M),
+};
+
+static int r8a779f0_cpg_enable_disable_core(const struct device *dev,
+					   struct cpg_clk_info_table *clk_info,
+					   uint32_t enable)
+{
+	int ret = -ENOTSUP;
+	uint32_t reg;
+
+	enable = !!enable;
```

Comment:
does not look like this is used, stale? guess you  may want to remove the argument entirely

Reply:
It's not used yet as current driver don't need their clocks to be enabled or disabled manually.
This whole function is a placeholder to keep skeleton from other renesas clock driver.
It's also already planned that the arm64 support of this soc (coming just after this PR, see #60252 ) will need some clocks to be enable/disabled for some drivers.

More specifically, this argument is just as a boolean to know if this function is called to enable or disable the concerned clock.

That's all reasons why this function has to be here but is not used yet.

Reply:
ok, keep the argument, drop this line? or mark it as `ARG_UNUSED`